tests/test_emnlp_conference.py

```diff
@@ -341,91 +340,6 @@ def test_submit_papers(self, test_client, client, openreview_client, helpers):
             'EMNLP/2023/Conference/Submission5/Authors'
         ]
 
-        ## withdraw submission
```

Comment:
instead of deleting this case, can you withdraw that paper using the delete edit and revert the change?

I would also like to test the email notifications.